packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -35,9 +49,94 @@ export const statuses = {
   DEACTIVATED: 'deactivated'
 }
 
-function Integrations() {
+const PaddedAlert = styled(Alert)`
```

Comment:
You can make use of < Stack /> go remove alot of these custom components

Reply:
Ok. Would address this as well.